frontend/packages/core/src/Resolver/index.tsx

```diff
@@ -48,9 +48,16 @@ interface ResolverProps {
   searchLimit: number;
   onResolve: (data: { results: object[]; input: object }) => void;
   variant?: "dual" | "query" | "schema";
+  apiPackage?: any;
```

Comment:
add comment, also is there a type we can provide to avoid the confusion of passing string of package name vs package obj itself?